decidim-core/lib/decidim/core/test/shared_examples/reports_examples.rb

```diff
@@ -59,6 +59,52 @@
   end
 end
 
+shared_examples "higher user role hides resource with comments" do
+  context "and the admin hides a resource with comments" do
+    let!(:comment) { create(:comment, body: "Dummy comment", commentable: reportable, author: user) }
+
+    before do
+      login_as user, scope: :user
+      Decidim::Ai::SpamDetection.create_reporting_user!
+    end
+    around do |example|
+      previous = Capybara.raise_server_errors
+
+      # Disabling server errors to that we can test page not found error.
+      Capybara.raise_server_errors = false
+      example.run
+      Capybara.raise_server_errors = previous
+    end
+
+    it "reports the resource" do
```

Comment:
If I understand correctly, this should be like this, right? 

```suggestion
    it "hides the resource" do
```